lib/cli/test/unit/Cardano/CLISpec.hs

```diff
@@ -839,8 +847,14 @@ spec = do
                 (newCliKeyScheme s)
                 (newCliKeyScheme s)
 
-        it "scheme == mapKey (fromHex . toHex) scheme"
-            $ property prop_roundtripCliKeySchemeKeyViaHex
+        it "scheme == mapKey (fromBS . toBS) scheme" $ property
+            $ propCliKeySchemeCodecRoundTrip keyByteStringCodec
```

Comment:
The notion of "roundtripping" is now made a bit more complicated when the roundtrip can depend on the values themselves.

I should have a closer look at these.